### ISSUE TITLE
Debug replication failure when using AWS_PROFILE

### ISSUE THREAD

**Issue by tallenaz** [sul-dlss/preservation_catalog#889]
See https://app.honeybadger.io/projects/54415/faults/38261789#notice-summary for the stack trace and details.

**Comment by atz** [comment]
So it isn't finding AWS_REGION.

**Comment by tallenaz** [comment]
From https://stackoverflow.com/questions/44433527/how-to-load-config-from-aws-config

```
export AWS_SDK_LOAD_CONFIG="true"
```

**Comment by tallenaz** [comment]
Honeybadger is saying the failure is at `return if s3_object.exists?`
https://app.honeybadger.io/projects/54415/faults/38274053#notice-comments

and according to issue 460 in the aws-sdk-ruby, 

```
We were getting the AWS::S3::Errors::BadRequest for every call to .exist?. Our bucket was in us-east-2, and was somehow working without specifying the region. Once we passed region into the S3 client, all was well.
```

To test this, we can try setting AWS_PROFILE, AWS_SDK_LOAD_CONFIG and AWS_REGION in the environment and see if we get past `s3_object.exists?`

**Comment by tallenaz** [comment]
We discovered that the `~/.aws/config` and `~/.aws/credentials` files both support named profiles, and each has a different syntax for doing so -- the `config` file requires a 'profile' keyword, like `[profile foo]`, where the `credentials` file forbids that keyword, like `[foo]`. Once we corrected the syntax of the files on stage, objects replicated successfully. What was probably happening is that the sdk was unable to read credentials for a region.
